lib/dx/loaders/dataloader.ex

```diff
@@ -49,18 +53,21 @@ defmodule Dx.Loaders.Dataloader do
   defp where(opts, []), do: opts
   defp where(opts, conditions), do: Keyword.put(opts, :where, {:all, conditions})
 
-  def init() do
+  def init(opts \\ []) do
     repo = config(:repo)
 
     # workaround for dataloader incompatibility with transactions
     #   -> https://github.com/absinthe-graphql/dataloader/issues/129#issuecomment-965492108
     run_concurrently? = not db_conn_checked_out?(repo)
 
-    source =
-      Dataloader.Ecto.new(repo,
-        query: &Dx.Ecto.Query.from_options/2,
-        async: run_concurrently?
-      )
+    ecto_opts = [
+      query: &Dx.Ecto.Query.from_options/2,
+      async: run_concurrently?,
+      repo_opts: opts[:repo_options] || [],
```

Comment:
Now that the options are neatly attributed to the loader (at least in my head), maybe we also support `repo_opts` as an alternative key, because that's what `Dataloader.Ecto.new/2` uses.

```suggestion
      repo_opts: opts[:repo_options] || opts[:repo_opts] || [],
```